client/js/components/user/CustomerSettings/CustomerSettingsSupport.vue

```diff
@@ -30,7 +30,7 @@
             class="u-mb-0_75"
             data-cy="contactTitle"
             :pattern="titlesInUsePattern"
-            :data-dp-validate-error="customerContact.title === '' ? 'error.name.required' : 'error.name.unique'"
+            :data-dp-validate-error="customerContact.title === '' ? Translator.trans('error.name.required') : Translator.trans('error.name.unique')"
```

Comment:
```suggestion
            :data-dp-validate-error="Translator.trans(customerContact.title === '' ? 'error.name.required' : 'error.name.unique')"
```

Reply:
I have committed and accepted your suggestion, but I ask me why `Translator.trans` must be for `customerContact.title`?
Trans keys here are: `'error.name.required'` & `'error.name.unique'`?

Reply:
We don't pass `customerContact.title` to `Translator.trans`, we use it as a condition to check which translation key to pass - `'error.name.required'` or `'error.name.unique'`. It's the same as before, but it saves us a `Translator.trans`.